script.skin.helper.service/plugin.py

```diff
@@ -46,8 +46,36 @@ def __init__(self):
             elif action == "LAUNCH":
                 xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
                 path = sys.argv[2].split("&path=")[1]
-                xbmc.executebuiltin("Action(Close)")
                 xbmc.executebuiltin(path)
+            elif action == "FOCUSANDCLICK":
+                #used as workaround to display local media in extendedinfo actorinfo
```

Comment:
Yes, indeed. query the database for a movie by actor:

PluginContent.py line 1119:

json_result = getJSON('VideoLibrary.GetMovies', '{ "filter": {"operator": "contains", "field": "actor", "value": "%s"}, "properties": [ %s ] }' %(name,fields_movies))